src/main/resources/assets/minecolonies/lang/en_us.json

```diff
@@ -57,263 +57,260 @@
-  "minecolonies.config.limittoonewarehousepercolony": "Warehouse limit",
-  "minecolonies.config.limittoonewarehousepercolony.comment": "Should there be at max 1 warehouse per colony?. [Default: true]",
-  "minecolonies.config.builderbuildblockdelay": "Builder block delay",
-  "minecolonies.config.builderbuildblockdelay.comment": "Delay after each block placement (Increasing it, increases the delay) [Default: 15]",
-  "minecolonies.config.blockminingdelaymodifier": "Mining delay",
-  "minecolonies.config.blockminingdelaymodifier.comment": "Delay modifier to mine a block (Decreasing it, decreases the delay) [Default: 500]",
-  "minecolonies.config.maxblockscheckedbybuilder": "Max checked blocks",
-  "minecolonies.config.maxblockscheckedbybuilder.comment": "Amount of blocks the builder checks (to decrease lag by builder). [Default: 1000]",
+  "minecolonies.config.builderinfiniteresources.comment": "Should the Builder and Miner build without resources? (This also turns off what they produce.) [Default: false]",
+  "minecolonies.config.limittoonewarehousepercolony": "Warehouse Limit",
+  "minecolonies.config.limittoonewarehousepercolony.comment": "Should there be a max of 1 warehouse per colony?. [Default: true]",
+  "minecolonies.config.builderbuildblockdelay": "Builder Block Delay",
+  "minecolonies.config.builderbuildblockdelay.comment": "Delay after each block placement (increasing it increases the delay). [Default: 15]",
+  "minecolonies.config.blockminingdelaymodifier": "Mining Delay",
+  "minecolonies.config.blockminingdelaymodifier.comment": "Delay modifier to mine a block (decreasing it decreases the delay). [Default: 500]",
+  "minecolonies.config.maxblockscheckedbybuilder": "Max Checked Blocks",
+  "minecolonies.config.maxblockscheckedbybuilder.comment": "Amount of blocks the Builder checks (to decrease lag). [Default: 1000]",
   "minecolonies.config.chatfrequency": "Chat Frequency",
   "minecolonies.config.chatfrequency.comment": "Chat frequency of worker requests. [Default: 30]",
   "minecolonies.config.enableindevelopmentfeatures": "Dev Features",
-  "minecolonies.config.enableindevelopmentfeatures.comment": "Should in development features be enabled (might be buggy). [Default: false]",
+  "minecolonies.config.enableindevelopmentfeatures.comment": "Should development features be enabled (might be buggy). [Default: false]",
   "minecolonies.config.alwaysrendernametag": "Name tags",
   "minecolonies.config.alwaysrendernametag.comment": "Should citizen name tags be rendered? [Default: true]",
   "minecolonies.config.growthmodifier": "Growth Modifier",
-  "minecolonies.config.growthmodifier.comment": "Child growth modifier, default on avg they take about 60min to grow (at 1.0x modifier). Setting to 5 = 5x as fast. [Default: 1]",
+  "minecolonies.config.growthmodifier.comment": "Child growth modifier. On average children take about 60min to grow at 1.0x modifier. 5=5x as fast. [Default: 1]",
```

Comment:
Hmm, I would not remove the connection between 60 mins and default value. Also spaces between mathematical symbols

Reply:
Fixed